src/tensors/cpu/wasm_intgemm_interface.h

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+/* Main interface for integer matrix multiplication (C = A * B) for wasm.
+ *
+ * A is typically activations whose rows should be a multiple of 1 (i.e. no restriction) and
+ * columns should be a multiple of 64.
+ * 
+ * B is typically fixed model parameters whose rows should be a multiple of 64 and columns
+ * should be a multiple of 8.
+ * 
+ * C is row major.
+ */
+
+#include <cstdint>
+
+using Index = unsigned int;
+
+// A structure holding 8-bit quantized values and the associated quantization parameters.
+typedef struct QuantizedBuffer {
+    int8_t* value;
+    float scale;
+    int8_t zero_point;
```

Comment:
@kpu  we want `float` zero_point, right?

Reply:
So that would be the float value that int8_t 0 corresponds to?  Then it may as well be:
int8_t *value;
float min_range /* -128 maps to this */, max_range; /* 127 maps to this */
Which is more flexible.  

I had been thinking 0 is always somewhere in the quantized range and the int8_t value tells us which one maps to 0.  Because so many values are near 0 it's usually important to have an exact 0 quantization point though it could be expressed in the above form.

Reply:
So the `zero_point` stays as `int8_t`?

Reply:
@kpu What is the final call on this one?

Reply:
@kpu I was thinking of, taking matrix A, subtracting a float value X, so that everything else is zero centered and then quantising, But in practise we just want to not care about outliers.

Reply:
@kpu A gentle ping 👍

Reply:
I am ok with @XapaJIaMnu 's more general proposal that zero_point be float.

Reply:
Great. Changing it to `float` then 👍

Reply:
@abhi-agg the zero point is not a `float` yet.

Reply:
Done.